abaco.go

```diff
@@ -133,15 +133,14 @@ func (group *AbacoGroup) updateFrameTiming(p *packets.Packet, frameIdx FrameInde
 	newSubframeCount := frameIdx * abacoSubframeDivisions
 	deltaSubframe := newSubframeCount - group.LastSubframeCount
 	deltaTs := ts.T - group.LastFirmwareTimestamp.T
+	if deltaSubframe <= 0 || deltaTs == 0 {
+		return
```

Comment:
I would suggest that we at least LOG the event, and maybe check that whatever values we care about other than the timestamp increase are the same between this packet and the last packet. If we're just getting 2 identical (for our purposes) packets, sure lets go ahead and ignore one. But if they disagree about what is happening, I am not comfortable going forward without better understanding. Eg what if one is right and one is wrong and our experimental results depend on picking the right one.